src/FileSystemUtilities.js

```diff
@@ -63,6 +64,12 @@ export default class FileSystemUtilities {
       }
       type = "file";
     }
+    if (pathIsAbsolute(src) && process.platform !== "win32") {
+      if (type === "junction") {
+        type = "dir";
+      }
```

Comment:
I think this bit is unnecessary?  The `type` argument is [ignored outside of windows](https://nodejs.org/api/fs.html#fs_fs_symlink_target_path_type_callback).

> The type argument can be set to 'dir', 'file', or 'junction' (default is 'file') and is only available on Windows (ignored on other platforms).